doc/util.md

```diff
@@ -241,3 +242,34 @@ Operators:
  - `&`: Combines two `Vector2`s to form a Rect, the origin should be on the left and the size on the
   right.
  - `%`: Modulo/Remainder of x and y separately of two `Vector2`s.
+
+### Matrix4
+
+This class comes from the `vector_math` package. We have created a few extension methods on top
+of what is already offered by `vector_math`.
+
+Methods:
+  - `transform2`: Create a new `Vector2` by transforming the given `Vector2` using the `Matrix4`.
+  - `transformed2`: Transform the input `Vector2` into the output `Vector2`.
+
+Getters:
+  - `m11`: Returns the value at the first row and first column.
+  - `m12`: Returns the value at the first row and second column.
+  - `m13`: Returns the value at the first row and third column.
+  - `m14`: Returns the value at the first row and fourth column.
+  - `m21`: Returns the value at the second row and first column.
+  - `m22`: Returns the value at the second row and second column.
+  - `m23`: Returns the value at the second row and third column.
+  - `m24`: Returns the value at the second row and fourth column.
+  - `m31`: Returns the value at the third row and first column.
+  - `m32`: Returns the value at the third row and second column.
+  - `m33`: Returns the value at the third row and third column.
+  - `m34`: Returns the value at the third row and fourth column.
+  - `m41`: Returns the value at the fourth row and first column.
+  - `m42`: Returns the value at the fourth row and second column.
+  - `m43`: Returns the value at the fourth row and third column.
+  - `m44`: Returns the value at the fourth row and fourth column.
```

Comment:
Maybe we could omit that `Returns the value at the` as it repeats on all items ?

Reply:
Documentation wise it makes sense to put it on each? As it explains what the getter does. Just saying first row second column is imo not saying anything

Reply:
idk, imo just saying `The first row...` is enough for, especially since this is aligned under the getters section.

Either is up to you, totally optional

Reply:
Oh wait I thought this was in the dart code... I have updated this as I do agree on that part